packages/MachineLearning/src/ValueObjects/AiCapabilityStatus.php

```
@@ -0,0 +1,109 @@
+
+        $reasonCodes = [];
+        foreach ($data['reason_codes'] as $reasonCode) {
+            if (!is_string($reasonCode) || trim($reasonCode) === '') {
+                throw AiRuntimeContractException::invalidValue('AI capability status reason codes');
+            }
+
+            $reasonCodes[] = $reasonCode;
+        }
+
+        if (!is_bool($data['available'])) {
+            throw AiRuntimeContractException::invalidValue('AI capability status availability');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\MachineLearning\Exceptions\AiRuntimeContractException' in method 'fromArray'. (undefined)

(StaticAccess)